libs/core/src/lib/alert/alert.component.scss

```diff
@@ -12,3 +13,9 @@
 .fd-has-display-none {
     display: none;
 }
+
+.fd-alert__close {
+    border-width: 1px;
+    border-style: solid;
+    border-color: transparent;
+}
```

Comment:
This should be coming from Fundamental-styles

Reply:
There are some issues with button classes, it's solved in https://github.com/SAP/fundamental-ngx/pull/1770